hammer/vlsi/driver.py

```diff
@@ -1165,13 +1168,14 @@ def par_output_to_timing_input(output_dict: dict) -> Optional[dict]:
                  or None if output_dict was invalid
         """
         try:
-            input_files = deeplist([output_dict["par.outputs.output_netlist"]])
+            input_files = deeplist([output_dict.get("par.outputs.output_physical_netlist", output_dict["par.outputs.output_netlist"])])
             result = {
                 "timing.inputs.input_files": input_files,
                 "timing.inputs.input_files_meta": "append",
                 "timing.inputs.top_module": output_dict["par.inputs.top_module"],
+                "timing.inputs.post_synth_sdc": output_dict["par.inputs.post_synth_sdc"],
                 "timing.inputs.spefs": output_dict["par.outputs.spefs"],
-                "timing.inputs.sdf_file": output_dict["par.outputs.sdf_file"],
```

Comment:
What about the case for post-P&R timing for non-hierarchical flows? Wouldn't you need to input an SDC file that's not the `post_synth_sdc`?